spec/requests/shops_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
```

Comment:
remove all this file.
We don’t have specs file yet

Reply:
done